clang/lib/Index/IndexUnitReader.cpp

```diff
@@ -43,14 +43,15 @@ class IndexUnitReaderImpl {
   bool IsSystemUnit;
   bool IsModuleUnit;
   bool IsDebugCompilation;
-  StringRef WorkingDir;
-  StringRef OutputFile;
-  StringRef SysrootPath;
+  SmallString<128> WorkingDir;
+  SmallString<128> OutputFile;
+  SmallString<128> SysrootPath;
```

Comment:
I believe the recommendation is usually to work on SmallString but then persist with `std::string` (see https://llvm.org/docs/ProgrammersManual.html#std-string). I realize there's already one `SmallString` here but... not need to make it worse I guess 😆